src/workers/postCommentedRedis.ts

```diff
@@ -14,7 +14,10 @@ const worker: Worker = {
     const data: Data = messageToJson(message);
     try {
       const postNotificatiion = await getPostNotification(con, data.postId);
-      await redisPubSub.publish('events.posts.commented', postNotificatiion);
+
+      if (postNotificatiion) {
+        await redisPubSub.publish('events.posts.commented', postNotificatiion);
+      }
```

Comment:
It would trigger `publish` with `undefined` which has no point

Reply:
Nice, we need more of these 😂